### PR TITLE
CuboCore.packages: added

### PR DESCRIPTION
**Pull request by dan4ik605743**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by dan4ik605743 — COMMENTED** [review]
image
All successfully

**Comment by dan4ik605743** [comment]
Removed an extra file, I forgot about it. Everything is also perfectly bite.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 133118` at 0b1f5754 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>22 packages built successfully:</summary>
  <ul>
    <li>CuboCore.coreaction</li>
    <li>CuboCore.corearchiver</li>
    <li>CuboCore.corefm</li>
    <li>CuboCore.coregarage</li>
    <li>CuboCore.corehunt</li>
    <li>CuboCore.coreimage</li>
    <li>CuboCore.coreinfo</li>
    <li>CuboCore.corekeyboard</li>
    <li>CuboCore.corepad</li>
    <li>CuboCore.corepaint</li>
    <li>CuboCore.corepdf</li>
    <li>CuboCore.corepins</li>
    <li>CuboCore.corerenamer</li>
    <li>CuboCore.coreshot</li>
    <li>CuboCore.corestats</li>
    <li>CuboCore.corestuff</li>
    <li>CuboCore.coreterminal</li>
    <li>CuboCore.coretime</li>
    <li>CuboCore.coretoppings</li>
    <li>CuboCore.coreuniverse</li>
    <li>CuboCore.libcprime</li>
    <li>CuboCore.libcsys</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Consider adding a comment explaining the purpose of this patch on the line preceeding.
Near pkgs/applications/misc/cubocore-packages/libcprime/default.nix:23:5:
```
   |
23 |     ./0001-fix-application-dirs.patch
   |     ^
```
</li>
  </ul>
</details>

**Comment by dan4ik605743** [comment]
@SuperSandro2000 please check my pr

File: pkgs/top-level/cubocore-packages.nix [hunk 1]
```diff
@@ -0,0 +1,93 @@
+{ newScope, lxqt, lib, libsForQt5 }:
```
Comment (SuperSandro2000) [inline pkgs/top-level/cubocore-packages.nix:1]: Please format using 2 spaces.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -12665,6 +12647,16 @@ in
 
   ### END OF LUA
 
+  ### CuboCore
+  CuboCore = recurseIntoAttrs (import ./cubocore-packages.nix {
+    inherit newScope;
+    inherit lxqt;
+    inherit lib;
+    inherit libsForQt5;
```
[the remark this answers is not in the document]
Comment (dan4ik605743) [reply pkgs/top-level/all-packages.nix]: Ok.

File: pkgs/top-level/cubocore-packages.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/top-level/cubocore-packages.nix:1]: Most of the lines below are not formatted with the right amount of spaces. You can use nixpkgs-fmt to get it right.

**Comment by dan4ik605743** [comment]
@SuperSandro2000 Ready, formatted Cubocore-Packages.nix using nixpkgs-fmt.

**Comment by dan4ik605743** [comment]
@SuperSandro2000 pls check my pr

**Comment by dan4ik605743** [comment]
@SuperSandro2000 pls check my pr

**Comment by dan4ik605743** [comment]
@SuperSandro2000 pls check my pr

**Comment by dan4ik605743** [comment]
@SuperSandro2000 pls check my pr

**Comment by dan4ik605743** [comment]
@SuperSandro2000 pls check

**Pull request closed without merging**